### PR TITLE
adding pinned dependancies

### PR DESCRIPTION
**Pull request by Emterry**
adding further pinned dependancies from scorecard remediations

### CODE REVIEW

File: .github/workflows/ci-devcontainer.yaml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
 
       - name: "Install devcontainer CLI"
         id: install_devcontainer_cli
-        run: npm install -g @devcontainers/cli
+        run: npm install -g @devcontainers/cli@0.268.0
```
Comment (jacobwoffenden) [inline .github/workflows/ci-devcontainer.yaml]: Latest is `0.30.0` https://www.npmjs.com/package/@devcontainers/cli?activeTab=versions

File: .github/workflows/ci-devcontainer.yaml [hunk 2]
```diff
@@ -73,7 +73,7 @@ jobs:
...
```
Comment (jacobwoffenden) [inline .github/workflows/ci-devcontainer.yaml]: Same as https://github.com/ministryofjustice/data-platform/pull/111/files#r1108588781

**Review by jacobwoffenden — CHANGES_REQUESTED** [review]
Will dependabot update these NPM packages?

File: .github/workflows/ci-devcontainer.yaml (hunk 1 above)
Comment (Emterry) [reply .github/workflows/ci-devcontainer.yaml]: thank you for spotting that i was mistakenly looking at : https://www.npmjs.com/package/@vscode/dev-container-cli

- i will change it now

**Comment by Emterry** [comment]
> Will dependabot update these NPM packages?

dependabot will update action workflows as desribed here: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot#enabling-dependabot-version-updates-for-actions

but im not sure if this extends to upgrading a specific command within a workflow

**Review by jacobwoffenden — APPROVED** [review]

**Review by julialawrence — APPROVED** [review]

**Pull request merged**